fedimint-core/src/module/audit.rs

```diff
@@ -63,50 +68,86 @@ impl Display for Audit {
 pub struct AuditItem {
     pub name: String,
     pub milli_sat: i64,
-    pub module_name: String,
+    pub module_metadata: Option<ModuleMetadata>,
+}
+
+impl AuditItem {
+    fn sats(&self) -> f64 {
+        (self.milli_sat as f64) / 1000.0
+    }
+
+    fn module_kind_or(&self, default: String) -> String {
+        self.module_metadata
+            .as_ref()
+            .map(|meta| meta.module_kind.to_string())
+            .unwrap_or(default)
+    }
+}
+
+pub struct ModuleMetadata {
+    pub module_kind: String,
```

Comment:
Saving the kind here every time seems redundant. Don't we have the instance->kind mapping available when querying the audit items? Injecting the kind after aggregating the audit items would be most elegant imo.

Reply:
I think this one would be nice to fix @bradleystachurski before merging. The other comment needs more thought and is out of scope.

Reply:
Great call, that's a much better approach 👍